### PR TITLE
feat: Added Image and Input Component

### PR DESCRIPTION
**Pull request by xii-vi**
# Image

 - ### Responsive Image
 - ### Round Image
 - ### Square Imag
Screenshot (23)

# Input

 - ### Standard Input
 - ### Validation and Error Inputs

Screenshot (25)

### CODE REVIEW

File: pages/image.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    <meta name="description" content="Images of different types available to use in your project">
+    <title>RA-One | Image</title>
+    <link rel="stylesheet" href="/stylesheet/main.css">
+</head>
+
+<body>
+    <div>
+        <nav>
+            <div class="logo">
+                <img>
```
Comment (sanjay-xdr) [inline pages/image.html:18]: you need this img tag or its by mistake?

**Review by sanjay-xdr — APPROVED** [review]
everthing looks good

File: pages/input.html [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    <meta name="description" content="Images of different types available to use in your project">
+    <title>RA-One | Image</title>
+    <link rel="stylesheet" href="/stylesheet/main.css">
+</head>
+
+<body>
+    <div>
+        <nav>
+            <div class="logo">
+                <img>
```
Comment (syed-shaida-hussain) [inline pages/input.html:17]: What is this img tag here?

File: pages/image.html (hunk 1 above)
Comment (xii-vi) [reply pages/image.html:18]: it is needed as I am planning to complete the navigation after the P0 Screens.

File: stylesheet/input.css [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+color: var(--dnd-color);
+}
+.input-success{
+border: 2px solid var(--online-color);
+color: var(--online-color);
+}
+.input-warning{
+border: 2px solid var(--warning-icon);
+color: var(--warning-icon);
+}
```
Comment (syed-shaida-hussain) [inline stylesheet/input.css:24]: try to name --warning-icon as --warning-icon-color or something like that because it is a color not an icon.

**Review by syed-shaida-hussain — CHANGES_REQUESTED** [review]
Do these minor changes . Rest is good.

File: stylesheet/input.css (hunk 3 above)
Comment (xii-vi) [reply stylesheet/input.css:24]: _Suggested changes are made_

File: pages/input.html [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+                <iframe
+                    src="https://carbon.now.sh/embed?bg=rgba%2814%2C107%2C168%2C1%29&t=material&wt=sharp&l=htmlmixed&width=&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Cascadia+Code&fs=13.5px&lh=168%25&si=false&es=2x&wm=false&code=%253Cinput%2520type%253D%2522text%2522%2520placeholder%253D%2522Error%2520message%2522%2520class%253D%2522mb-2%2520p-2%2520input%2520input-error%2522%253E%250A%253Cinput%2520type%253D%2522text%2522%2520placeholder%253D%2522Warning%2520message%2522%2520class%253D%2522mb-2%2520p-2%2520input%2520input-warning%2522%253E%250A%253Cinput%2520type%253D%2522text%2522%2520placeholder%253D%2522Success%2520message%2522%2520class%253D%2522mb-2%2520p-2%2520input%2520input-success%2522%253E"
+                    sandbox="allow-scripts allow-same-origin">
+                </iframe>
+                <footer class="mt-4">
+                    <a href="/pages/image.html">&lt Image</a>
+                    <a href="/pages/input.html">Input &gt</a>
+                </footer>
+            </div>
+            <div class="right-side-container">
```
Comment (amirnathani9) [inline pages/input.html]: use proper naming convention

**Review by amirnathani9 — COMMENTED** [review]
Do the suggested changes. Everything looks good

File: pages/input.html (hunk 4 above)
Comment (xii-vi) [reply pages/input.html]: Suggested changes are made

**Review by amirnathani9 — COMMENTED** [review]
Looks Good

**Pull request merged**